GeosCore/gc_environment_mod.F90

```diff
@@ -142,6 +142,13 @@ SUBROUTINE GC_Allocate_All( Input_Opt,                        &
        ' -> at GC_Allocate_All  (in module GeosCore/gc_environment_mod.F90)'
 
 #ifdef FASTJX
+    ! Throw an error if FAST-JX is used for simulations other than Hg
+    IF ( Input_Opt%ITS_A_FULLCHEM_SIM .or. Input_Opt%ITS_A_CARBON_SIM ) THEN
```

Comment:
This makes an assumption of future simulations that will use photolysis. A more robust test is check if .NOT. Input_Opt%ITS_A_MERCURY_SIM.